conf/i18n/en/my-account.yaml

```diff
@@ -0,0 +1,66 @@
+myAccount:
```

Comment:
With issue - I meant ticket. ;)

right, the translations are also in the theme.